src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py

```diff
@@ -0,0 +1,213 @@
+from base64 import b64encode
+import os
+import requests
+from requests.utils import to_native_string
+
+from azure.cli.core.prompting import prompt, prompt_pass, NoTTYException
+import azure.cli.core.azlogging as azlogging
+from azure.cli.core.util import CLIError
+
+from ._utils import (
+    get_resource_group_name_by_registry_name,
+    get_registry_by_name
+)
+from .credential import acr_credential_show
+
+logger = azlogging.get_az_logger(__name__)
+APIUSERNAME = os.getenv('multiUser')
+APIPASSWORD = os.getenv('multiPass')
+
+class NotFound(Exception):
+    pass
+
+
+class Unauthorized(Exception):
+    pass
+
+def _basic_auth_str(username, password):
+    return 'Basic ' + to_native_string(b64encode(('%s:%s' % (username, password)).encode('latin1')).strip())
+
+def _headers(username, password):
+    auth = _basic_auth_str(username, password)
+    return {'Authorization': auth, 'Content-Type': 'application/JSON'}
+
+def get_response_for_show(resource_group_name, registry_name, name, api_username, api_password):
+    registry_name, _ = get_registry_by_name(registry_name, resource_group_name)
+    id = registry_name.id
+    endpoint = "http://api20170808022712.azurewebsites.net" + id + "/buildDefinitions/" + name
+    response = requests.get(endpoint, headers=_headers(api_username, api_password))
```

Comment:
If your response is already json, you can return `requests.get ...` directly and let the CLI handle the result.